admin_app/src/components/ContentModals.tsx

```diff
@@ -139,6 +146,36 @@ export const EditModal: React.FC<EditModalProps> = ({
               }}
             />
           </div>
+          <div className="relative px-5 py-1 flex flex-auto items-center">
+            <label
+              htmlFor="content_language"
+              className="block text-gray-700 text-sm font-bold mr-2"
+            >
+              Language
+            </label>
+            <div className="relative inline-block">
+              <select
+                id="content_language"
+                className="shadow appearance-none border active:outline-none border-neutral-400 text-sm rounded w-32 dark:bg-gray-800 py-2 px-3 text-gray-800 dark:text-gray-400 overflow-auto required mt-2"
+                defaultValue={cardToEdit?.content_language || "ENGLISH"}
```

Comment:
"ENGLISH" is hard coded here. What about adding it to default value for `cardToEdit.content_language`?

Reply:
This may also lead to some tricky bug. Ideally, we should have strict requirements of `cardToEdit` here. If `content_language` is null, it should throw an error.

Reply:
Hmm I was thinking about this but for cardToEdit, we have 2 states:
1. No cardToEdit (it is null) because we're creating a new content
2. There is a cardToEdit but somehow the language is by default "ENGLISH"
I am not sure how to implement 2.

I am trying to also define "card being edited type" where null values are possible if unset, and "card to be saved / saved card" where null values are not allowed.